src/main/java/hudson/plugins/git/GitSCM.java

```diff
@@ -891,6 +911,7 @@ public String getGitExe(Node builtOn, EnvVars env, TaskListener listener) {
         return null;
     }
 
+    @SuppressFBWarnings(value="SE_BAD_FIELD", justification = "If your project or run aren't serializable you've got bigger problems")
```

Comment:
@oleg-nenashev Maybe better phrased as "marshallable" -- I'll make the change to avoid confusion.